src/components/chats/Chat.tsx

```diff
@@ -0,0 +1,83 @@
+import styled, { css } from "styled-components"
+import { FlexBox } from "components/layout/FlexBox"
+import { TextArea } from "components/forms/textfields/TextArea"
+import { Text } from "components/text/Text"
+import { Button } from "components/forms/Button"
+import { useTheme } from "styles/theme/useTheme"
+
+export type ChatProps = {
+  header?: React.ReactNode
+  chatBody: React.ReactNode
+  footer?: React.ReactNode
+  value?: string
```

Comment:
I didnt realize what this `value` prop was for at first. It may be confusing here since the entire chat box wouldn't have a `value` on its own. But the user input box would. So could you adjust the name to indicate what the value is for? Same with the onChange and onClick props below.